includes/class-sensei-course.php

```diff
@@ -2811,29 +2810,6 @@ public static function flush_rewrite_rules( $post_id ){
 
     }
 
-    /**
-     * Optionally return the full content on the single course pages
-     * depending on the users course_single_content_display setting
-     *
-     * @since 1.9.0
-     * @param $excerpt
-     * @return string
-     */
-    public static function full_content_excerpt_override( $excerpt ){
```

Comment:
We shouldn't remove this function outright as third party developers may be using it. Instead, we should [deprecate it](sensei_lesson_preview_title). Since the setting is no longer available, simply returning `$excerpt` would be the proper implementation for the function.

Reply:
Hi @donnapep , 

Excellent point there, i made the corresponding changes. I used as reference https://github.com/Automattic/sensei/pull/1869/commits/9b44a7eb5a164061c012bc1fde2331adfd4c6b8d to deprecate method `full_content_excerpt_everride();`